target/linux/ar71xx/image/Makefile

```diff
@@ -17,6 +17,7 @@ include ./tp-link.mk
 include ./ubnt.mk
 include ./generic.mk
 include ./legacy-devices.mk
+include ./senao.mk
```

Comment:
@mkresin @pepe2k suggested it to be a good idea to separate it out (currently this is only the ens202ext board, but I've got board support code set up also for the ens200 which I'll be pr'ing once I get this merged, and a host of other EnGenius devices which are identical as far as firmware format/layout is concerned).